doc/source/release-notes/Percona-Server-8.0.15-6.rst

```diff
@@ -43,12 +43,111 @@ New Features
 
 - RocksDB WAL file information can now be seen in the
   :table:`performance_schema.log_status` :ref:`table <log_status>`.
+
+- New :variable:`Audit_log_buffer_size_overflow` status variable has been
+  implemented to track when an :ref:`audit_log_plugin` entry was either
+  dropped or written directly to the file due to its size being bigger
+  than :variable:`audit_log_buffer_size` variable.
   
 
 Bugs Fixed
 ================================================================================
 
+- TokuDB and MyRocks native partitioning would never release memory which could
```

Comment:
I don't think this is correct: the issue was that memory was allocated from a wrong memroot, where objects had wrong lifetime and also became corrupted on multithreaded access. @satya-bodapati please help with better description here

Reply:
Right, it is not only about ever growing memory. So I would suggest

TokuDB and MyRocks native partitioning handler objects allocated from wrong MEM_ROOT. Memory is released only on shutdown and concurrent access to global MEM_ROOT caused memory corruptions and therefore crashes. 

Feel free to reword if necessary.

Is this enough? Should we describe how we fixed? @laurynas-biveinis

Reply:
@satya-bodapati LGTM. The focus in RNs should be in user terms